### PR TITLE
Fix collada textures not being loaded

### PR DESCRIPTION
**Pull request by achim-k**
**User-Facing Changes**
Fix collada textures not being loaded

**Description**
We were revoking temporary object URLs to early, resulting in errors when loading these URLs. This PR ties the lifetime of these temporary object URLs to the lifetime of the ModelCache.

### CODE REVIEW

**Review by snosenzo — APPROVED** [review]
Seems fine, though I do want to make sure: there's no problem with these models only being cleaned up on panel unmount? They shouldn't grow over time otherwise?

**Comment by achim-k** [comment]
> Seems fine, though I do want to make sure: there's no problem with these models only being cleaned up on panel unmount? They shouldn't grow over time otherwise?

No I don't think that there is an issue, especially since models are cached, so temporary object URLs are created only once for every model URL

File: packages/studio-base/src/panels/ThreeDeeRender/ModelCache.ts [hunk 1]
```diff
@@ -220,25 +219,21 @@ export class ModelCache {
         const objectUrl = URL.createObjectURL(
           new Blob([textureAsset.data], { type: textureAsset.mediaType }),
         );
-        textureUrls.set(textureUrl, objectUrl);
+        this.#colladaTextureObjectUrls.set(textureUrl, objectUrl);
```
Comment (jtbandes) [inline packages/studio-base/src/panels/ThreeDeeRender/ModelCache.ts:222]: Should we check if one already exists before creating a new one here?

Comment (achim-k) [reply packages/studio-base/src/panels/ThreeDeeRender/ModelCache.ts:222]: Good remark, I'll create a follow-up PR for this

**Pull request merged**